dev-python/wurlitzer/wurlitzer-2.0.0.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Capture C-level stdout/stderr in Python "
+HOMEPAGE="https://github.com/minrk/wurlitzer"
```

Comment:
https://github.com/minrk/wurlitzer/blob/master/test.py

Also add pypi page here.

Reply:
> https://github.com/minrk/wurlitzer/blob/master/test.py

I have added tests support, just `distutils_enable_tests pytest wasn't enough. I had to manually specify the `test.py` file for some reason.